src/style.css

```
@@ -0,0 +1,38 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  display: flex;
+  height: 16px;
+  width: 16px;
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  background-image: url(images/star-active.svg);
+}
+
+.stars--0 :nth-child(n + 1) {
+  background-image: url(images/star.svg);
+}
+
+.stars--1 :nth-child(n + 2) {
+  background-image: url(images/star.svg);
+}
```

```suggestion
.stars--0 :nth-child(n + 1),
.stars--1 :nth-child(n + 2) {
  background-image: url(images/star.svg);
}
```
you may join selectors with same styles (as you did before) - FYI

---

Ok, right. Have implemented! TYSM!